### DISCUSSION TITLE
Dell U3219Q not detected on USB-C

### DISCUSSION THREAD

**Discussion by dannycpw** [osy86/HaC-Mini#587] [Q&A]
Hi All,

My NUC8I7HVK can't detect U3219Q on USB-C most of the time (including boot & after boot), even I tried to plug & unplug. Sometimes even HDMI can't be detected while connected before boot up and need to unplug then re-plug it in.

Can anyone give me idea for how to fix it?

Configuration

    HaC Mini version : 3.0.1
    OSX version : 11.3.1
    NUC model : NUC8I7HVK
    Boot SSD model and protocol : Samsung SSD 970 EVO Plus 1TB NVMe (primary / OSX / APFS)
    Second SSD model and protocol : KXG60ZNV1T02 NVMe KIOXIA 1024GB NVMe (2nd / APFS)
    First DIMM model : 16GB HMA82GS6AFR8N-UH
    Second DIMM model : 16GB HMA82GS6AFR8N-UH
    Display port(s) used : USB-C, HDMI
    Display resolution(s) : 4k
    Monitor : Dell U3219Q (firmware: M3B107)
    Any TB3 devices: no
    Any USB devices: Dell unified usb receiver for keyboard / mouse; Wacom intuos-pro
    Any Wi-Fi/Bluetooth devices : Intel 8265

Thanks & regards,

Danny

**Comment by osy** [comment]
Since you mention on boot, I'm assuming you mean you can't see the Intel BIOS screen or setup? Can you boot into Windows or Ubuntu?

**Comment by Medizinmann-MD** [comment]
Did you try to switch the display off/on directly before booting?

I need to to this with my AOC Agon AG271QG - while my LG LG 27" 27HJ712C and EIZO-Displays don't need ist....

...and another idea - what BIOS-Version do you use?

**Comment by dannycpw** [comment]
BIOS ver. 2.2.25

Power:
my nuc8 & mon are connected to same PDU. I do not switch off my mon after shutdown nuc8 & let it goes into powersave mode. And then cut off the PDU supply, normally.
When I need to boot up the nuc8, I switch on the mains of the PDU, the monitor go up to power save mode, then power on the nuc8.
If monitor is unable to be detected, I tried to soft switch on / off the monitor (on the monitor panel), but not help. However, unplug / plug-in the USB-C sometimes may help.

Boot:
The "Intel" logo can be shown while booting up with hdmi connection & able to load boot setup, boot device select menu, etc.
Also, I'm able to boot into Win10 (Win2Go) without any issue.

I am doing some research and wonder that ppl claimed using usb-c to DP (alt mode) can resolve the problem... & seems like Dell only support such connection for those usb-c monitor. I will give it a try & looking for a right cable. Is that mean this "unable to be detected" issue is due to the support of Dell monitor? Sorry, I'm a bit confuse if the issue is form Dell or OSX... just thinking if we may force  OSX to use DP mode for usb-c to usb-c cable connection, but no clue yet.

BTW, my "Hardware Video Codec" is on "Intel" & the Thunderbolt is already on "native" & also patched NVN firmware.

**Reply by osy** [reply]
If it works in Windows then the problem isn’t the monitor. Do you know what resolution macos is set to? Maybe it’s unsupported?

**Reply by dannycpw** [reply]
i'm using "Scaled (3360 x 1890)", will try native 4k (3840 x 2160) by "Default for display".
BTW, my OC enabled "Retina display boot".

**Reply by dannycpw** [reply]
just tested, same result. it can't be detected even unplug then re-plug.
